### DISCUSSION TITLE
AASA (apple-app-site-association) support

### DISCUSSION THREAD

**Discussion by pnewell** [teamhanko/hanko#427] [Q&A]
Is there a built-in way to add a well-known route for AASA?

**Comment by FlxMgdnz** [comment]
Not yet. Since we're currently offering web components and not a hosted login page, it's not on our immediate roadmap, as the relying party would have to host the aasa on their own page. 

We're working on example apps for iOS and Android, though, and will probably make those work together with our current example apps.
